src/lib/export_org.js

```diff
@@ -216,15 +216,23 @@ export const attributedStringToRawText = parts => {
     .join('');
 };
 
-export default (headers, todoKeywordSets) => {
+export default (headers, todoKeywordSets,fileConfigLines) => {
```

Comment:
Minor nit, but would you mind installing Prettier (and probably prettier-emacs), @andersjohansson? It'll make little spacing issues like this a complete non-issue.

Reply:
Thanks, everything should be prettyfied now.